### PR TITLE
create team by ThuBM and DucNV

### PR DESCRIPTION
**Pull request by DucNguyenVan**
create team by ThuBM and DucNV using react

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ gem "jbuilder", "~> 2.0"
 gem "react-rails", "~> 1.0"
 gem "factory_girl_rails", "~> 4.0"
 gem "sdoc", "~> 0.4.0", group: :doc
+gem 'bootstrap-datepicker-rails'
```
Comment (NguyenHai2310) [inline Gemfile]: ""

File: app/assets/javascripts/admin/projects.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (NguyenHai2310) [inline app/assets/javascripts/admin/projects.coffee]: remove

File: app/assets/javascripts/components/team_form.js.coffee [hunk 3]
```diff
@@ -0,0 +1,96 @@
+@TeamForm = React.createClass
+  getInitialState: ->
+    name: ''
```
Comment (NguyenHai2310) [inline app/assets/javascripts/components/team_form.js.coffee]: "

File: app/assets/stylesheets/admin/projects.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the admin/projects controller here.
+// They will automatically be included in application.css.
```
Comment (NguyenHai2310) [inline app/assets/stylesheets/admin/projects.scss]: remove

File: app/assets/stylesheets/custom.css.scss [hunk 5]
```diff
@@ -271,3 +271,14 @@ span.picture {
...
+.date {
+  margin-bottom: 0px;
+  padding-bottom: 0px;
+  padding-top: 0px;
+}
+
+.members-list {
+  height: 400px !important;
+  overflow-y: scroll;
+}
```
Comment (NguyenHai2310) [inline app/assets/stylesheets/custom.css.scss]: :(

File: app/controllers/admin/projects_controller.rb [hunk 6]
```diff
@@ -1,6 +1,27 @@
...
+    @project = Project.new
+    @teams = Team.all
+    @members = User.all
+    if request.xhr?
+      render json: Team.find_by(params[:team_id]).members
+    end
+  end
+
+  def create
+    # byebug
```
Comment (NguyenHai2310) [inline app/controllers/admin/projects_controller.rb]: remove

File: app/helpers/admin/projects_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module Admin::ProjectsHelper
+end
```
Comment (NguyenHai2310) [inline app/helpers/admin/projects_helper.rb]: remove

File: app/views/admin/projects/_member_form.html.erb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+<%= f.label member_form.name %>
+<%= f.check_box :members %>
```
Comment (NguyenHai2310) [inline app/views/admin/projects/_member_form.html.erb]: :(

File: app/controllers/admin/teams_controller.rb [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  def new
+    @team = Team.new
+  end
+
+  def create
+    @team = Team.create team_params
+    if @team.save
+      render json: @team
+    else
+      render "new"
```
Comment (phonghenry14) [inline app/controllers/admin/teams_controller.rb]: :new

File: app/views/admin/projects/new.html.erb [hunk 10]
```diff
@@ -0,0 +1 @@
+<%= react_component "Project", {teams: @teams}  %>
```
Comment (phonghenry14) [inline app/views/admin/projects/new.html.erb]: thừa space

File: app/views/admin/teams/index.html.erb [hunk 11]
```diff
@@ -0,0 +1 @@
+<%= react_component "Teams", {teams: @teams, users: @users}  %>
```
Comment (phonghenry14) [inline app/views/admin/teams/index.html.erb]: thừa space

**Pull request merged**